python_modules/libraries/dagster-dbt/dagster_dbt/__init__.py

```diff
@@ -12,7 +12,12 @@
     DbtCliResource as DbtCliResource,
     dbt_cli_resource as dbt_cli_resource,
 )
-from .cli.resources_v2 import DbtManifest as DbtManifest
+from .cli.resources_v2 import (
+    DbtClientV2 as DbtClientV2,
+    DbtCliEventV2 as DbtCliEventV2,
```

Comment:
I don't think we should export these as top level exports until after things become a bit less experimental (and we'll want to rename these regardless)